samples/expert-irb-adjusted.rb

```diff
@@ -79,16 +79,16 @@ def run(str)
     when "\n"
       begin
         out, obj = IRBalike.run(@cmd + ';')
-        @str += ["#@cmd\n",
+        @str += ["#{@cmd}\n",
```

Comment:
Didn't even realize this was allowed with strings! Glad to see it going away :grin: